shard.yml

```diff
@@ -1,5 +1,5 @@
 name: router
-version: 0.1.13
+version: 0.3.0
```

Comment:
Why is the version 0.2.0 skipped?

Reply:
Because of https://github.com/tbrand/router.cr/releases, the version seems to be in reality `0.2.1`